rpm/pom.xml

```diff
@@ -45,7 +45,13 @@
             <artifactId>junit-jupiter-params</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <!-- show log output during tests using log4j as the slfj backend. -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
```

Comment:
I know, this is just my disliking of log4j … could you use logback instead.

Reply:
I have switched to https://mvnrepository.com/artifact/ch.qos.logback/logback-classic V 1.2.3 and pushed.